zrp/zrp.py

```diff
@@ -96,14 +96,45 @@ def rename_data_columns(self, data):
         # self.params_dict = {}
         return data
     
-    def transform(self, input_data):
+    def delete_old_files(self):
```

Comment:
Could lead to accidental deletions, adding a message warning the user to provide a new directory path might be better than file deletion.

Reply:
Good idea – I changed the code as suggested